src/sparseml/pytorch/utils/exporter.py

```diff
@@ -655,9 +655,9 @@ def is_match(node: onnx.NodeProto) -> bool:
         # find any node in the graph that uses the output of `node`
         # as an input. replace the input with `node`'s input
         for other in model.graph.node:
-            for i in range(len(other.input)):
+            for i, other_input_i in enumerate(other.input):
```

Comment:
Could we rename `i` ?

Reply:
why?

Reply:
I'm good either way, our codebase tries to use something like `index` over `i` where ever we can

for index: 31 matches in 20 files
for i: 29 matches in 16 files

So it's upto you!

Reply:
I choose `i` 😁

Reply:
Accepted!